Example/Emission/ARRootViewController.m

```diff
@@ -147,6 +149,16 @@ - (ARCellData *)jumpToWorksForYou
   }];
 }
 
+
+- (ARCellData *)jumpToConsignments
+{
+  return [self tappableCellDataWithTitle:@"Start Consignment Flow" selection:^{
+    id viewController = [[ARComponentViewController alloc] initWithEmission: nil moduleName:@"Consignments" initialProperties: @{}];
```

Comment:
not created a root view controller for it yet, will give it a bit later once I know what initial props etc could be